src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+[...document.querySelectorAll('li')].forEach(li => {
+  const span = document.createElement('span');
+  if (li.parentNode.querySelector('ul')) {
+    li.prepend(span);
+    span.prepend(span.nextSibling);
+  }
+});
+
+tree.addEventListener('click', e => {
+  if (e.target.tagName !== 'SPAN') {
+    return;
+  }
+
+  const childrenContainer = e.target.parentNode.querySelector('ul');
+
+  if (!childrenContainer) {
+    return;
+  }
+
+  childrenContainer.style.display = (childrenContainer.style.display === 'none') ? 'block' : 'none';
```

Comment:
Don't forget about linter checks. Fix it 
![image](https://github.com/mate-academy/js_sliding-tree-DOM/assets/70009205/49067be5-00bc-442e-a43a-b70b0ce6d47a)